<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testResult type="org.overture.typechecker.tests.ClassesPpTypeCheckTest">
<message column="9" line="2376" message="Definition 'exp' not used" messageType="warning" number="5000" resource="CodegenPP"/>
<message column="14" line="2380" message="Definition 'type' not used" messageType="warning" number="5000" resource="CodegenPP"/>
<message column="8" line="2394" message="Recursive function has no measure" messageType="warning" number="5012" resource="CodegenPP"/>
<message column="13" line="2524" message="Definition 'd' not used" messageType="warning" number="5000" resource="CodegenPP"/>
<message column="13" line="2523" message="Definition 'c' not used" messageType="warning" number="5000" resource="CodegenPP"/>
<message column="54" line="2140" message="Polymorphic function has not been instantiated" messageType="error" number="3350" resource="CodegenPP"/>
<message column="12" line="2156" message="Operation returns void value" messageType="error" number="3313" resource="CodegenPP"/>
<message column="12" line="2170" message="Operation returns void value" messageType="error" number="3313" resource="CodegenPP"/>
<message column="12" line="2180" message="Operation returns void value" messageType="error" number="3313" resource="CodegenPP"/>
<message column="12" line="2197" message="Operation returns void value" messageType="error" number="3313" resource="CodegenPP"/>
<message column="9" line="2338" message="Impure operation '(e.hasDeflt)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="5" line="2356" message="Impure operation '(letExp.getDefs)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="41" line="2388" message="Impure operation '(selif.getTest)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="33" line="2388" message="Name Compile(?) is ambiguous" messageType="error" number="3010" resource="CodegenPP"/>
<message column="33" line="2388" message="Cannot access Compile(?) from a static context" messageType="error" number="3181" resource="CodegenPP"/>
<message column="41" line="2389" message="Impure operation '(selif.getThn)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="33" line="2389" message="Name Compile(?) is ambiguous" messageType="error" number="3010" resource="CodegenPP"/>
<message column="33" line="2389" message="Cannot access Compile(?) from a static context" messageType="error" number="3181" resource="CodegenPP"/>
<message column="43" line="2390" message="Impure operation '(selif.getElif)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="60" line="2390" message="Impure operation '(selif.getEse)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="9" line="2392" message="Cannot use 'new' in a functional context" messageType="error" number="3348" resource="CodegenPP"/>
<message column="9" line="2397" message="Cannot access Compile(SimpleExpression) from a static context" messageType="error" number="3181" resource="CodegenPP"/>
<message column="49" line="2400" message="Impure operation '(head.getTest)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="41" line="2400" message="Name Compile(?) is ambiguous" messageType="error" number="3010" resource="CodegenPP"/>
<message column="41" line="2400" message="Cannot access Compile(?) from a static context" messageType="error" number="3181" resource="CodegenPP"/>
<message column="49" line="2401" message="Impure operation '(head.getThn)' cannot be called from here" messageType="error" number="3300" resource="CodegenPP"/>
<message column="41" line="2401" message="Name Compile(?) is ambiguous" messageType="error" number="3010" resource="CodegenPP"/>
<message column="41" line="2401" message="Cannot access Compile(?) from a static context" messageType="error" number="3181" resource="CodegenPP"/>
<message column="12" line="2403" message="Cannot use 'new' in a functional context" messageType="error" number="3348" resource="CodegenPP"/>
<result/>
</testResult>
